cosmos/spring-cloud-azure-starter-data-cosmos/spring-cloud-azure-data-cosmos-aks-sample/pom.xml

```diff
@@ -4,17 +4,17 @@
   <modelVersion>4.0.0</modelVersion>
   <parent>
     <groupId>com.azure.spring</groupId>
-	<artifactId>azure-spring-boot-samples</artifactId>
-	<version>1.0.0</version>
-	<relativePath>../../../pom.xml</relativePath>
+	  <artifactId>azure-spring-boot-samples</artifactId>
```

Comment:
It's still `tab` here. Please use `space` instead.
![image](https://user-images.githubusercontent.com/13167207/203196567-e955983d-c574-4368-a801-51dafebbd4bf.png)